CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/).
+
+## [0.0.1]
```

Comment:
```suggestion
## [unreleased]
```

(if we add a version to the Changelog we should also bump the gem version in the same PR, but I'll change it again as we only bump in master)